src/tests/MyEvents.test.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import MyEvents from '../componentsTest/MyEvents';
+
+describe('<myEvents />', () => {
+
+  it('loads my events on first render', async () => {
+    global.fetch = jest.fn(() => {
+      Promise.resolve();
+    })
+    const loadGetMyEvents = jest.fn().mockName('loadGetMyEvents');
+    render(<MyEvents loadGetMyEvents={loadGetMyEvents} />);
+    expect(loadGetMyEvents).toHaveBeenCalled();
```

Comment:
please add at least two more tests, you can reference my tests here: https://github.com/Safewaan/HonkedIn/pull/32/files#diff-fe907acee125d459ad5da3e1199c880c55a93c33998cf659b79ab55389c567e8

note how they're the same tests with minor differences (check for b instead of a).

Reply:
i did npm i run and did all that, made 2 more but this error: will push for now but only 1/3 tests pass
<img width="534" alt="Screenshot 2023-02-24 at 4 20 11 PM" src="https://user-images.githubusercontent.com/76895894/221295260-1aeb2445-67b8-46d4-9a83-82c0e2c4c619.png">